lib/test-data.json

```diff
@@ -88,8 +88,8 @@
   ],
   "edges": [
     {
-      "source": "a",
-      "target": "g",
+      "source": "0+a",
```

Comment:
I see. It's not scoped in the Vertex object in the JSON. But it's scoped in the sense of the graph's GCL structure.